src/release.js

```diff
@@ -67,6 +68,11 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
     if (!adaptees[subjectType]) {
         throw new Error(`No implementation found for the type '${subjectType}'`);
     }
+
+    if ( releaseVersion && semverValid(releaseVersion) === null) {
```

Comment:
(Consistency) Any reason for testing `=== null` here when you chose to only test general truthiness of `semverValid()` in the previous usage?